content/documentation/overview/index.adoc

```diff
@@ -238,6 +242,70 @@ This section lists all the validations that Kiali performs on all Istio configur
 |link:/files/validation_examples/401.yaml[401.yaml, window="_blank"]
 |===
 
+=== Istio Wizards
+
+Kiali provides different actions to create, update and delete Istio configuration driven by Wizards.
+These actions are located under Service Details page.
+
+.Service Details Actions
+image::service-istio-actions.png[Service Istio Actions]
+{nbsp} +
+These actions are enabled by default. +
+Kiali can be also installed in "view only" mode to restrict any write operation on Istio configuration. +
+Check https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml#L134[Kiali Operator CR] to get more details about how to configure this option.
```

Comment:
Please add option window="_blank" to open this in a new tab
`[Kiali Operator CR, window="_blank"]`